packages/cli/src/commands/gsn-relayer-withdraw.ts

```diff
@@ -0,0 +1,44 @@
+import { CommandsLogic, WithdrawOptions } from '../CommandsLogic'
+import { gsnCommander, getKeystorePath, getServerConfig } from '../utils'
+import { createCommandsLogger } from '../CommandsWinstonLogger'
+import { KeyManager } from '@opengsn/relay/dist/KeyManager'
+import { toBN } from 'web3-utils'
+
+const commander = gsnCommander(['f', 'g'])
+  .option('-k, --keystore-path <keystorePath>', 'relay manager keystore directory', '/app/data/gsndata/manager/')
+  .option('-s, --server-config <serverConfig>', 'server config file', '/app/data/config/gsn-relay-config.json')
+  .option('-b, --broadcast', 'broadcast tx after logging it to console', false)
+  .requiredOption('-a, --amount <amount>', 'amount of funds to withdraw to owner address, in wei')
+  .parse(process.argv);
+
+(async () => {
+  const config = getServerConfig(commander.serverConfig)
+  const host = config.ethereumNodeUrl
+  const logger = createCommandsLogger(commander.loglevel)
+  const logic = await new CommandsLogic(host, logger, {}).init()
+  const keystorePath = getKeystorePath(commander.keystorePath)
+  const keyManager = new KeyManager(1, keystorePath)
+
+  const withdrawOptions: WithdrawOptions = {
+    withdrawAmount: toBN(commander.amount),
+    keyManager,
+    config,
+    broadcast: commander.broadcast
+  }
+  console.log('config is', config)
+  console.log('withdrawalAmount is', withdrawOptions.withdrawAmount.toString())
+  console.log('broadcast is', withdrawOptions.broadcast)
+  const result = await logic.withdrawToOwner(withdrawOptions)
+  if (result.success) {
```

Comment:
AFAIK, cli should wait for mining, and if you want to be explicit, a cli item `--nowait` to avoid waiting for mining..
displaying the TX is good thing anyway.

Reply:
Since it's only one tx that's being sent, I don't see a reason to wait for mining. Most people will open etherscan anyway to see that this hash is there and mined.